feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/App.kt

```diff
@@ -228,3 +230,7 @@ val randomApp
       ),
       developerName = getRandomString(range = 2..5, capitalize = true)
     )
+
+fun App.isInCatappult(): Boolean? {
```

Comment:
You don't have to create an extension if you are literally in the same file.
Just create a member function inside `App` itself 🙂 
Just to avoid unnecessary separate imports for the extension fun...